From f11699bb802013a5488f2fa2d0cfe834c9f01355 Mon Sep 17 00:00:00 2001
From: Vineesh k k <vineesh.k.k@intel.com>
Date: Mon, 18 May 2015 17:42:56 +0530
Subject: [PATCH] EM/ACPI: ACPI entry for Wireless charging

Added bios device entry names in the acpi_platform file.
This is required for NX2A4WP Wireless power receiver
on CHT_HR platform. GPIOs are used for controlling the
Nordic chip.

Change-Id: I6eadb8dc75b3a23ba73747cc4effc083c31d444e
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1817
Signed-off-by: Vineesh k k <vineesh.k.k@intel.com>
---
 drivers/acpi/acpi_lpss.c     | 2 +-
 drivers/acpi/acpi_platform.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/acpi/acpi_lpss.c b/drivers/acpi/acpi_lpss.c
index f0701a839ce7..c887bbd220d5 100644
--- a/drivers/acpi/acpi_lpss.c
+++ b/drivers/acpi/acpi_lpss.c
@@ -213,7 +213,7 @@ static const struct acpi_device_id acpi_lpss_device_ids[] = {
 	{ "INT3437", },
 	{ "INT3496", },
 	{ "GPTC0001", },
-
+	{ "INTA4321", },
 	/* BYT PWM */
 	{ "80860F09", },
 	{ }
diff --git a/drivers/acpi/acpi_platform.c b/drivers/acpi/acpi_platform.c
index c0e03b9a45d1..01e94cb14f04 100644
--- a/drivers/acpi/acpi_platform.c
+++ b/drivers/acpi/acpi_platform.c
@@ -67,7 +67,7 @@ static const struct acpi_device_id acpi_platform_device_ids[] = {
 	{ "HSP0001", 0 },
 
 	{ "INT33A2" },
-
+	{ "INTA4322" },
 	{ }
 };
 
-- 
1.9.1

